5333/src/main/cpp/Robot5333.cpp

```diff
@@ -23,8 +23,13 @@ void Robot::RobotInit() {
   DrivetrainConfig drivetrainConfig{robotmap.drivetrain.LeftGearbox, robotmap.drivetrain.RightGearbox};
   drivetrain = new Drivetrain(drivetrainConfig);
 
-  ElevatorConfig elevatorConfig{ robotmap.lift.ElevatorGearbox, nullptr, nullptr, 2.1, 25 / 1000.0, 20 };
-  beElevator = new Lift(elevatorConfig);
+  harvesterSolenoid = new DoubleSolenoid(0, 1);
+
+  HarvesterIntakeConfig harvesterConfig{ *harvesterGearbox, *harvesterSolenoid };
```

Comment:
HarvesterGearbox is never created, at the moment this program will crash on the RoboRIO

Reply:
Ugh I don't have time to fix it rn; tomorrow it is